src/App.css

```diff
@@ -0,0 +1,12 @@
+@import url(/Users/amritesh/Downloads/Coding/react/react-trainee/src/index.css);
```

Comment:
Hey @ammrit2312 don't use absolute paths of your machine, any images or svgs that you want to use, create a copy of them in the assets folder and use relative paths of the directory.
Don't go out of the scope of the directory.

Reply:
This is to prevent any issues when we change from one device to another and also helps in the deployment